chukwuemekachm/M-Tracker-React-Client
# #161065956 Completes admin view all requests

#### What does this PR do?
- This PR completes admin view all requests
#### Description of Task to be completed?
- Modify Dashboard.jsx
- Add UserDetails.jsx
- Add adminRequestActions.js
- Modify existing tests
- Modify requestsReducer.js
- Update snapshots
#### How should this be manually tested?
- Clone the repo
- Start up command line
- Navigate to the project directory using `cd`
- Run `npm install`
- Run `npm start` or `npm run dev`
- Your default browser should be opened if you used `npm run dev`
- Login or signup(navigate to `/login` or `/signup`)
- You will be redirected to the dashboard
- Observe the dashboard
- Click on any request and preview it
#### Production URL
- [heroku](https://m-tracker-react.herokuapp.com)
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- [#161065956](https://www.pivotaltracker.com/story/show/161065956)
#### Screenshots (if appropriate)

<img width="1440" alt="screen shot 2018-10-09 at 4 06 47 am" src="https://user-images.githubusercontent.com/33798252/46644995-9ae5cc00-cb7a-11e8-9ac5-39cc676b8482.png">


<img width="1440" alt="screen shot 2018-10-09 at 4 07 29 am" src="https://user-images.githubusercontent.com/33798252/46644996-9b7e6280-cb7a-11e8-8590-23e23f3620ac.png">


<img width="1440" alt="screen shot 2018-10-09 at 4 06 35 am" src="https://user-images.githubusercontent.com/33798252/46644994-9ae5cc00-cb7a-11e8-9115-89d038762807.png">


#### Questions:
- None
